pkg/api/server/cel2sql/convert.go

```diff
@@ -20,9 +20,14 @@ import (
 	"github.com/google/cel-go/cel"
 )
 
-// Convert takes CEL expressions and attempt to convert them into Postgres SQL
-// filters.
-func Convert(env *cel.Env, filters string) (string, error) {
+// ConvertView takes a View and CEL expressions and attempt to convert them into
+// Postgres SQL filters.
+func ConvertView(view *View, filters string) (string, error) {
```

Comment:
I wouldn't change the name of the function. Ultimately, what is being converted is the CEL expression and not the view, which seems to be merely a container to represent the mapping among the database table and the object.

Reply:
👍